Documentation/namespace-metrics.md

```diff
@@ -4,4 +4,5 @@
 | ---------- | ----------- | ----------- |
 | kube_namespace_status_phase| Gauge | `namespace`=&lt;ns1&gt; <br> `status`=&lt;Active\|Terminating&gt; |
 | kube_namespace_labels | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_LABEL`=&lt;NS_LABEL&gt; |
+| kube_namespace_annotations | Gauge | `namespace`=&lt;ns1&gt; <br> `label_NS_LABEL`=&lt;NS_LABEL&gt; |
```

Comment:
`s/_LABEL/_NAMESPACE`

Reply:
BTW, maybe we should add a `kubeAnnotationsToPrometheusLabels` and make the prefix for  annotations being `annotation_`. WDYT @brancz

Reply:
yeh wasn't sure if a new function made sense, but happy whatever the decision @andyxning